src/Type/NullType.php

```diff
@@ -115,6 +115,16 @@ public function getIterableValueType(): Type
 		return new ErrorType();
 	}
 
+	public function isOffsetAccesible(): TrinaryLogic
+	{
+		return TrinaryLogic::createNo();
```

Comment:
This is technically not true. as you can do `null[123]` or `null[new stdClass]`. I'm not sure how many people rely on this behavior. Similar is true of booleans, but AFAIK `null[123]` is used a lot more in real world than `true[123]`.

I'd recommend keeping the No result and seeing how many stuff will it break in real world.

Reply:
what is this shit? :shit: